dev-resources/logback-test.xml

```diff
@@ -5,10 +5,6 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="trace"/>
-    <logger name="log4j.logger.org.eclipse.jetty.server" level="warn"/>
-    <logger name="log4j.logger.org.eclipse.jetty.util.log" level="warn"/>
-
     <root level="error">
```

Comment:
I don't see what purpose `logback.xml` serves, at this point.  It seems like it could be deleted.

I wouldn't worry about consistency too much beyond the contents of `logback-test.xml`.  Other repositories like puppetserver have a dev TK config file for use in the REPL or via `lein run`.  Since this repo is just a set of middleware, it doesn't have anything like that.